session/src/test/java/org/apache/iotdb/session/IoTDBSessionComplexIT.java

```diff
@@ -594,6 +608,25 @@ private void rawDataQuery() throws StatementExecutionException, IoTDBConnectionE
     sessionDataSet.closeOperationHandle();
   }
 
+  private void lastDataQuery() throws StatementExecutionException, IoTDBConnectionException {
+    List<String> suffixPath = new ArrayList<>();
+    List<String> prefixPath = new ArrayList<>();
+
+    suffixPath.add("s1");
+    suffixPath.add("s2");
+    prefixPath.add("root.sg1.d2");
+
+    SessionDataSet sessionDataSet = session.executeLastDataQuery(suffixPath, prefixPath);
+    sessionDataSet.setFetchSize(1024);
+
+    int count = 0;
+    while (sessionDataSet.hasNext()) {
+      count++;
+    }
+    Assert.assertEquals(2, count);
```

Comment:
As we control the insert of the records, it would be better to add one assert that can make sure the value of the two measurements.